contracts/universalSchemes/GlobalConstraintRegistrar.sol

```diff
@@ -35,16 +35,11 @@ contract GlobalConstraintRegistrar is UniversalScheme {
         bytes32 voteToRemoveParams; // Voting parameters for removing this GC.
     }
 
-    // Struct holding the data for each organization
-    struct Organization {
-        bytes32 voteRegisterParams; // The voting parameters for adding a GC.
-        IntVoteInterface intVote; // The voting machine in which the voting takes place.
-        mapping(bytes32=>GCProposal) proposals; // A mapping from the proposal ID to the proposal itself.
-        mapping(address=>bytes32) voteToRemoveParams; // A mapping that saves the parameters for removing each GC.
-    }
+    // GCProposal by avatar and proposalId
```

Comment:
Not sure what we gain having this modification.

Reply:
Removes unused code and adds the ability to access proposals. See #466

Reply:
how about access proposal by organizationsData[_avatar].proposals[_proposalId] ?
Which code is unused ?

Reply:
`Organization.voteRegisterParams` and `Organization.IntVoteInterface intVote`

Note this change also makes `voteToRemoveParams` available, which it currently isn't.

Reply:
> how about access proposal by organizationsData[_avatar].proposals[_proposalId] 

This is explained in #466.  The proposals mapping is not exposed, not accessible.

Reply:
please add tests for this change.

Reply:
Done